Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandGroupMemberKick.java

```diff
@@ -77,6 +77,14 @@ public final void run(final Map<String, String> parameters)
 		{
 			throw new UserErrorMessage("The creator cannot be kicked you fool.");
 		}
+		
+		// If the target is an admin and the current player is not the group creator,
+		if (kickCharacter.getProperty("groupStatus").equals("Admin") &&
+				!(GameUtils.equals((Key)admin.getKey(),
+						(Key)group.getProperty("creatorKey"))))
+		{
+			throw new UserErrorMessage("You cannot kick a group member of the same rank.");
```

Comment:
Since there is no mechanic for rank, better to use different wording so to not confuse the noobs.